lib/taiko.js

```diff
@@ -906,6 +906,7 @@ module.exports.title = title;
  * @param {number} [options.elementsToMatch=10] - Number of elements to loop through to match the element with given selector.
  * @param {string[]} [options.waitForEvents = ['firstMeaningfulPaint']] - Events available to wait for ['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint', 'targetNavigated']
  * @param {number} [options.waitForStart=100] - time to wait for navigation to start. Accepts time in milliseconds.
+ * @param {boolean} [options.force=false] - Set force to true to force doing action irrespective of element being unactionable(invisible, disabled etc)
```

Comment:
Is it ok to shorten the message as follows?
Can we also add example code of passing this option?

```suggestion
 * @param {boolean} [options.force=false] - Set to true to perform action on hidden/disabled elements
```

Reply:
Done 👍